.github/workflows/Component.BuildTest.yml

```diff
@@ -20,7 +20,7 @@ on:
         required: false
         type: string
       os-list:
-        default: '[ "windows-latest", "ubuntu-latest" ]'
+        default: '[ "windows-latest", "ubuntu-latest", "otel-linux-arm64" ]'
```

Comment:
Where do you get this OS "otel-linux-arm64"? I didn't find it with a simple search.

Reply:
@xiang17, these runners are used in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/. No problems so far.